restore-postgres-backup/action.yml

```diff
@@ -22,6 +22,12 @@ inputs:
   backup-file:
     description: Name of the backup file to restore. Must include the .gz extension if the remote file has it.
     required: true
+  restore-env:
```

Comment:
These 2 inputs are only used for the summary. Can we not add them and use only the required inputs? app-name and resource group are sufficient to identify the environment.

Reply:
removed from input